# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/udev/hotfix-clang.patch
# Copyright (C) 2023 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- udev-182/src/udevd.c.vanilla	2023-05-12 19:14:18.991033064 +0200
+++ udev-182/src/udevd.c	2023-05-12 19:14:26.001033570 +0200
@@ -47,6 +47,8 @@
 #include "udev.h"
 #include "sd-daemon.h"
 
+bool udev_builtin_validate(struct udev *udev);
+
 static bool debug;
 
 void udev_main_log(struct udev *udev, int priority,
